frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -156,6 +163,13 @@ end
 
 function ReaderTypeset:onSetBlockRenderingMode(mode)
     self:setBlockRenderingMode(mode)
+    local rendering_text = {
+        _("legacy"),
+        _("flat"),
+        _("book"),
+        _("web"),
+    }
```

Comment:
I will look into this later, maybe when I go to koptoptoins. Right now, I don't know how to start with this :/

Reply:
Looks like Dispatcher is already building a list with most infos about each thingy in cre/koptions:
https://github.com/koreader/koreader/blob/3d713dc34c0f568f1a028eaa951cd5c92da1176e/frontend/dispatcher.lua#L308-L317
so it can use it to populate the gesture actions menu:
https://github.com/koreader/koreader/blob/3d713dc34c0f568f1a028eaa951cd5c92da1176e/frontend/dispatcher.lua#L506-L510

So, may be you can add a method to it, so in the event handlers, one can use
```lua
Dispatcher:getOptionText("block_rendering_mode", mode)
Dispatcher:getOptionText("smooth_scaling", toggle)
```
Pinging @yparitcher : is this is a viable solution? I'm not familiar with the lifecycle of Dispatcher.